src/games/strategy/triplea/ui/TripleaMenu.java

```diff
@@ -410,10 +403,37 @@ public void actionPerformed(final ActionEvent e) {
     });
     menuGame.add(mapZoom).setMnemonic(KeyEvent.VK_Z);
   }
+  
+  private void addUnitNationDrawMenu(final JMenu parentMenu){
+    final JMenu unitSizeMenu = new JMenu();
+    unitSizeMenu.setMnemonic(KeyEvent.VK_N);
+    unitSizeMenu.setText("Show Unit Nation");
+    final ButtonGroup unitNationGroup = new ButtonGroup();
+    final JRadioButtonMenuItem nationOption0 = new JRadioButtonMenuItem("Hide Nation");
+    final JRadioButtonMenuItem nationOption1 = new JRadioButtonMenuItem("Next to Unit");
+    final JRadioButtonMenuItem nationOption2 = new JRadioButtonMenuItem("Below Unit");
```

Comment:
To be clear, the 5 lines are, one for the new RadioButtonMenu Item, then 4 more to do the action listener.